radiospectra/net/sources/psp.py

```diff
@@ -0,0 +1,137 @@
+import astropy.units as u
+from sunpy.net import attrs as a
+from sunpy.net.dataretriever.client import GenericClient, QueryResponse
+from sunpy.util.scraper import Scraper
+
+__all__ = ['RFSClient']
+
+RECEIVER_FREQUENCIES = {
```

Comment:
I would wonder if anyone would want to be able to search for `rfs_lfr` and `rfs_hfr` rather than by value of wavelength?

Reply:
Yea it's like that in all the sunpy clients so I thought it was necessary, I'll remove and see what happens

Reply:
I could add them as detectors maybe?

Reply:
yeah I think that may work. I'm not too sure about people using RSF - but given the the files are given separately I imagine people will want the transparency of searching by lfr and hfr - what do you think?

Reply:
Created a new issue for this #45